core/include/detray/builders/grid_factory.hpp

```diff
@@ -181,6 +214,16 @@ class grid_factory {
         constexpr auto e_z_axis = static_cast<dindex>(axes_t::label1);
 
         auto b_values = grid_bounds.values();
+        scalar_type min_phi = -constant<scalar_type>::pi;
+        scalar_type max_phi = constant<scalar_type>::pi;
+        if (axis_spans[0].size() > 1) {
+            min_phi = axis_spans[0][0];
```

Comment:
Are these used anywhere? Also, I think, the cylinder currently cannot observe any phi boundaries

Reply:
Removed.